core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -457,6 +459,9 @@ object GameStarter {
         freeTiles: MutableList<TileInfo>,
         startScores: HashMap<TileInfo, Float>
     ): TileInfo {
+        if (gameSetupInfo.gameParameters.noStartBias) {
+            return freeTiles.random()
```

Comment:
and this makes me wonder.. shouldn't the last return also be .random() instead of .last() to choose a random tile instead of always spawning you on the last tile in the list?